src/main/webapp/mentors.js

```diff
@@ -12,7 +15,57 @@ function initForm() {
 }
 
 /**
- * Get mentors from the mentor servlet.
+ * User Object
+ * @typedef {Object} User
+ * @property {string[]} interestTags - The user's tags
+ * @property {string[]} projectIDs - The user's projects
+ * @property {string} bio - The bio of the user
+ * @property {string} email - the email address of the user
+ * @property {string} gitHubID: - The user's github page
+ * @property {string} image - The User's profile picture
+ * @property {string} location - the location of the user
+ * @property {string} name - the name of the user
+ */
+
+/**
+ * Populate mentor search page with mock mentors.
+ */
+function getMockMentors() { // eslint-disable-line no-unused-vars
+  const mentors = [
+    {
+      interestTags: ['code', 'trapeze', 'baking'],
+      projectIDs: ['opensesame'],
+      bio: 'Sami works on opensesame',
+      email: 'samialves@google.com',
+      image: 'images/dior.jpg',
+      gitHubID: 'Sami-2000',
+      name: 'Sami Alves',
+      location: 'Mansfield, MA',
+    },
+    {
+      interestTags: null,
+      projectIDs: null,
+      bio: null,
+      email: 'obiabbi@google.com',
+      image: 'images/dior.jpg',
+      gitHubID: 'Obinnabii',
+      name: 'Obi Abii',
+      location: 'Idk maybe near Cornell',
+    },
+  ];
+  const mentorsContainer = document.getElementById('mentors-container');
+  if (!mentorsContainer) {
+    return;
+  }
+  mentorsContainer.innerHTML = '';
+  for (const mentor of mentors) {
```

Comment:
do we need `getMockMentors` anymore now that all of the data is being stored/retrieved from datastore?